Sources/PostgresNIO/New/PostgresRow-multi-decode.swift

```diff
@@ -3,7 +3,11 @@
 extension PostgresRow {
     @inlinable
     @_alwaysEmitIntoClient
-    public func decode<T0: PostgresDecodable, JSONDecoder: PostgresJSONDecoder>(_: (T0).Type, context: PostgresDecodingContext<JSONDecoder>, file: String = #file, line: Int = #line) throws -> (T0) {
+-n     public func decode<T0: PostgresDecodable
+-n , JSONDecoder: PostgresJSONDecoder>(_: (T0
+-n ).Type, context: PostgresDecodingContext<JSONDecoder>, file: String = #file, line: Int = #line) throws
+-n  -> (T0
+) {
```

Comment:
Why does echo suddenly print the `-n`. @gwynne do you have an idea why this might be happening?